src/highlight.cpp

```diff
@@ -10,7 +10,7 @@
 using namespace synth;
 
 
-unsigned const kMaxRefRecursion = 16;
+static unsigned const kMaxRefRecursion = 16;
```

Comment:
`const` at namespace level already implies `static` in C++. See http://stackoverflow.com/a/178259/2128694.